yeonjae22/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/view/product/ProductActivity.kt

```diff
@@ -7,63 +7,35 @@ import androidx.recyclerview.widget.LinearLayoutManager
 import com.ElevenStreetApi
 import com.exam.RetrofitInstance
 import com.exam.elevenstreet.data.ProductLocalDataSource
+import com.exam.elevenstreet.data.ProductRemoteDataSource
+import com.exam.elevenstreet.data.ProductRepository
 import com.example.elevenstreet.ProductResponse
-import com.example.elevenstreet.ProductXmlPullParserHandler
 import kotlinx.android.synthetic.main.activity_main.*
-import java.net.URL
-
 
 class ProductActivity : AppCompatActivity() {
-
-    private var elevenStreetApi: ElevenStreetApi? = null
     val adapter = ProductAdapter()
 
-    interface CallBack {
-        fun onSuccess(productList1: List<ProductResponse>)
-        fun onFailure(message: String)
-    }
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
```

Comment:
```suggestion
        setContentView(R.layout.activity_product)
```
Class명 변경에 맞추어 수정해주세요.

Reply:
수정했습니다